ext/standard/basic_functions_arginfo.h

```diff
@@ -82,3 +82,23 @@ ZEND_END_ARG_INFO()
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_crc32, 0, 1, IS_LONG, 0)
 	ZEND_ARG_TYPE_INFO(0, str, IS_STRING, 0)
 ZEND_END_ARG_INFO()
+
+#if defined(HAVE_SYSLOG_H)
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_openlog, 0, 3, _IS_BOOL, 0)
```

Comment:
Wait, are we generating `#if defined(HAVE_SYSLOG_H)` for all the 3 arginfos? Can't we generate just one?

@cmb69 @nikic

Reply:
Guess we could, but what's the problem with the current approach?